packages/create-cloudflare/e2e-tests/helpers.ts

```diff
@@ -490,6 +491,24 @@ export const testDeploymentCommitMessage = async (
 	expect(projectLatestCommitMessage).toContain(`framework = ${framework}`);
 };
 
+export async function testGitCommitMessage(
```

Comment:
Maybe add comments here - one liner is probably enough

Reply:
a2d8f89fcfbdac55b36978321feadcf75d8590af